src/gui/layout/qgslayoutitemguiregistry.h

```diff
@@ -509,6 +526,17 @@ class GUI_EXPORT QgsLayoutItemGuiRegistry : public QObject
      */
     void typeAdded( int metadataId );
 
+    /**
+     * Emitted whenever an item type is removed from the registry, with the specified
+     * \a metadataId.
```

Comment:
```suggestion
     * Emitted whenever an item type is removed from the registry, with the specified
     * \a metadataId.
     * \since QGIS 4.0
```